tests/scripts/run-xunit-perf.py

```diff
@@ -0,0 +1,454 @@
+import json
+import argparse
+import os
+import shutil
+import subprocess
+import sys
+import glob
+
+##########################################################################
+# Argument Parser
+##########################################################################
+
+description = 'Tool to run coreclr perf tests'
+
+parser = argparse.ArgumentParser(description=description)
+
+parser.add_argument('-testBinLoc', dest='coreclrPerf', default=None, required=True)
+parser.add_argument('-arch', dest='arch', default='x64', choices=['x64', 'x86'])
+parser.add_argument('-os', dest='operatingSystem', default=sys.platform, choices=['Windows_NT', 'Ubuntu16.04', 'Ubuntu14.04', 'OSX', sys.platform])
```

Comment:
>sys.platform [](start = 133, length = 12)

Do we need this?

Reply:
We need it in particular for Linux. When we push to benchview, we need to know what operating system in particular we are on (Ubuntu16.04, rather than simply Linux).

Reply:
Sorry, I meant to ask if `sys.platform` is needed? And if we need a default?

Reply:
Ah, I think we should have a default, and I thought sys.platform would be a better option than just saying Windows_NT.

Reply:
Actually, I guess the only place the variable gets used is uploading to Benchview... so I'm not sure.